content/episodes/2019-02-18-S04E06-decentralize-it.md

```diff
@@ -0,0 +1,244 @@
+---
+date: 2019-02-18T04:55:00-08:00
+title: "Decentralize It"
+ogtitle: "Decentralize It"
+description: |
+    Some people believe that decentralization is the inevitable future of the web. They believe that internet users will start to demand more privacy and authenticity of information online, and that they’ll look to decentralized platforms to get those things. But would decentralization be as utopian as advocates say it could be?
+
+    Host [Manoush Zomorodi](https://twitter.com/manoushz) speaks to Eugen Rochko of [Mastodon](https://joinmastodon.org/), an ad-free alternative to Twitter; Justin Hunter of [Graphite docs](https://app.graphitedocs.com/), a decentralized alternative to GoogleDocs; [Maria Bustillos](https://twitter.com/mariabustillos) who hopes to help eliminate fake news online through the Blockchain; [David Irvine](https://twitter.com/metaquestions?lang=en), the co-founder of [MaidSafe](https://maidsafe.net/) who plans to make the centralized internet as we know it redundant; and [Tom Simonite](https://twitter.com/tsimonite/) of [WIRED](https://www.wired.com/), who comments on both the promise and also the pitfalls of decentralization.
+ogdescription: "Explore the premise, the promise, and the pitfalls of decentralization — the antidote to Big Tech."
+number: 29
+season: 4
+seasonepisode: 6
+url: /season4/episode6/
+embed: "857f7522"
+mp3: "https://dts.podtrac.com/redirect.mp3/audio.simplecast.com/857f7522.mp3"
+categories: "episodes"
+host: "Manoushz"
+shownotes: |
+    Help us dream up the next season of IRL. What topics should we cover? Who should we talk to? [Let us know by filling out this survey](https://qsurvey.mozilla.com/s3/IRL-Listener-Survey).
+
+    Try out the decentralized endeavors covered in this episode of IRL:
+
+    - [Mastodon](https://mastodon.social/invite/Usw6HEoB)
+    - [Graphite Docs](https://app.graphitedocs.com/)
+    - [MaidSafe](https://maidsafe.net/)
+    - [Popula](https://popula.com/)
+
+    Decentralization efforts are proof that the age of internet innovation is far from over. In fact, Mozilla staff work tirelessly on decentralized web standards, which have been — and continue to be — widely adopted.
+
+    Mozilla co-chaired the [W3C Social Web Working Group](https://www.w3.org/wiki/Socialwg#Social_Web_Working_Group) 2014 through 2018, which produced several key decentralized social web standards. Some have dozens of implementations like:
+
+    - [Webmention](https://indieweb.org/Webmention) (a standard for federating conversations across the
+    decentralized web); and
+    - [MicroPub](https://indieweb.org/Micropub) (a standard API for client applications to post to decentralized web services).
+
+    Check out [IndieWeb.org](https://spec.indieweb.org/) for more on key decentralized web
```

Comment:
This links to the technical specs, should it go to the indieweb.org home page?

Reply:
The sentence is talking about the web standards specs but the wording is confusing. I've added a second link to try to make sure users end up where they expect to.